services/gateway/apis/http-version1/methods/delegates.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
 		status: {
 			optional: true,
 			type: 'string',
-			pattern: /^(?:\b(?:active|standby|banned|punished|non-eligible)\b|\b(?:active|standby|banned|punished|non-eligible|,){3,}\b){1}$/,
+			pattern: /^(?:\b(?:active|standby|banned|punished|ineligible)\b|\b(?:active|standby|banned|punished|ineligible|,){3,}\b){1}$/,
```

Comment:
This change probably needs to be reverted. @MichalTuleja please confirm.

Reply:
It might run into same compatibility issues as the APIv2 change. There's no need to revert it, however it is important to test the APIv2 with clients.